src/main/kotlin/apply/ui/api/ApplicationFormController.kt

```diff
@@ -0,0 +1,44 @@
+package apply.ui.api
+
+import apply.application.ApplicationFormSaveRequest
+import apply.application.ApplicationFormService
+import apply.application.ApplicationFormUpdateRequest
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+
+@RestController
+@RequestMapping("/api/register/application")
+class ApplicationFormController(
+        private val applicationFormService: ApplicationFormService
+) {
+    @GetMapping("/{id}")
+    fun getApplicationForm(@PathVariable id: Long): ResponseEntity<Any> {
+        return try {
+            val form = applicationFormService.getForm(1L, id)
+            ResponseEntity.ok().body(form)
+        } catch (e: NoSuchElementException) {
+            ResponseEntity.badRequest().body(e.message)
+        }
+    }
```

Comment:
```suggestion
    fun getApplicationForm(@PathVariable id: Long): ResponseEntity<ApplicationFormResponse> {
        return try {
            val form = applicationFormService.getForm(1L, id)
            ResponseEntity.ok().body(form)
        } catch (e: NoSuchElementException) {
            ResponseEntity.badRequest().body(e.message)
        }
    }
```

위와 같이 수정하니 아래 catch문에서 string을 던지기에 타입이 맞지 않아 에러가 나는군요.

글로벌 익셉션 핸들러의 필요성이 느껴지는 대목입니다..!
오늘 회의 때에 논의해보면 좋을 것 같아요~

오 아래에도 todo로 적어두셨군요😁